godot-core/src/docs.rs

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Could you not do a `#![cfg(since_api = "4.3")]` on the entire file, instead of individual symbols?

Reply:
but you can use `since_api` in `godot-macros` to change what is output by `derive(GodotClass)`

Reply:
Unresolved...

Reply:
What do you mean? look at godot-core/src/lib.rs...

Reply:
Please state this. When reading the comments on the conversation page, I don't see the diff, so it would help knowing which points are addressed and how. Thanks!